internal/bgp/frr/frr_test.go

```diff
@@ -59,7 +59,7 @@ func testSetup(t *testing.T) {
 }
 
 func testCheckConfigFile(t *testing.T) {
-	time.Sleep(10 * time.Millisecond) // give time to the debouncer to write the file
+	time.Sleep(100 * time.Millisecond) // give time to the debouncer to write the file
```

Comment:
sleeps make me sad.  What about making sure the timestamp on the file has been updated?

Reply:
:-) There should be no files, so we can even check if the file appears after a longer timeout.